### DISCUSSION TITLE
Filter on images

### DISCUSSION THREAD

**Discussion by charlesroper** [hluk/CopyQ#1679] [Q&A] [answered]
Is it possible to filter on images only? I've tried looking in the help docs and searching for "images" or "img" but it looks like I can only search for text within a clip, not for the clip type.

Many thanks!

**Comment by hluk** [comment, marked as answer]
You can search for `image/.*`. It matches items that contain given format, another example is `text/html` would filter HTML items.

**Reply by tyilo** [reply]
The problem with this is that it also matches any text containing `image/foo`. Is there no way to filter just on the mime type?
